instructions-for-devs.md

```diff
@@ -0,0 +1,199 @@
+  --db.host                     Hostname for SQL database               [string]
+  --db.port                     Port for SQL database                   [number]
+  --db.username                 User for SQL database                   [string]
+  --lnd.certpath                Path to the SSL certificate for lnd     [string]
+  --lnd.disable                 Disable lnd integration                [boolean]
+  --lnd.host                    Host of the lnd gRPC interface          [string]
+  --lnd.macaroonpath            Path of the admin macaroon for lnd      [string]
+  --lnd.port                    Port of the lnd gRPC interface          [number]
+  --p2p.listen                  Listen for incoming peers              [boolean]
+  --p2p.port, -p                Port to listen for incoming peers       [number]
+  --raiden.disable              Disable raiden integration             [boolean]
+  --raiden.port                 Port for raiden REST service            [number]
+  --rpc.port, -r                RPC service port                        [number]
+  --webproxy.disable            Disable web proxy server               [boolean]
+  --webproxy.port               Port for web proxy server               [number]
+```
+
+## 4. Interact with `xud` via Command-Line Interface
+
+Interact with an `xud` process, identified by its `rpc` host and port using `xucli`. For getting up-to-date CLI commands, use `./xucli --help` as shown below:
+
+```bash
+~/xud/bin$ ./xucli --help
+xucli <command>
+
+Commands:
+  xucli cancelorder <pair_id> <order_id>    cancel an order
+  xucli connect <host> [port]               connect to an xu node
+  xucli disconnect <host> [port]            disconnect from an xu node
+  xucli executeSwap <identifier> <role>     execute an atomic swap
+  <sending_amount> <sending_token>
+  <receiving_amount> <receiving_token>
+  xucli getinfo                             get general info from the xud node
+  xucli getorders [pair_id] [max_results]   get orders from the order book
+  xucli getpairs                            get order book's available pairs
+  xucli placeorder <pair_id> <order_id>     place an order
+  <quantity> [price]
+  xucli shutdown                            gracefully shutdown the xud node
+  xucli subscribepeerorders                 subscribe to incoming peer orders
+  xucli subscribeswaps                      subscribe to executed swaps
+
+Options:
+  --help          Show help                                            [boolean]
+  --version       Show version number                                  [boolean]
+  --rpc.port, -p  RPC service port                      [number] [default: 8886]
+  --rpc.host, -h  RPC service hostname           [string] [default: "localhost"]
+
+
+```
+
+Examples:
+```bash
+./xucli connect xud1.test.exchangeunion.com 8885
+# Manually connect to another xud instance
+
+./xucli placeorder BTC/LTC 1337 1 99
+#Places a new limit order BUYING 1 BTC for 99 LTC, assigning the orderId 1337
```

Comment:
I would prefer if there is a space between the comment itself and the `#`

Reply:
No need for nice words, I just overlooked it ;) - fixed.